### PR TITLE
Feature/dio 2434

### PR DESCRIPTION
**Pull request by mrpharderwijk**

### CODE REVIEW

File: src/organisms/dorel-section-content-block.html [hunk 1]
```diff
@@ -113,11 +121,29 @@ <h2 class="title">[[ blockData.title ]]</h2>
...
+       * image left text right
+       *
+       * @param {boolean} desktop breakpoint active or not
+       * @param {string} index of the repeated item
+       *
+       * @return {boolean}
+       */
       _reverseLayout(desktopBreakpoint, componentCount) {
+        console.log(desktopBreakpoint, componentCount);
         var imageLeft = !!(componentCount && componentCount % 2 !== 0);
```
Comment (aronmartin) [inline src/organisms/dorel-section-content-block.html]: const

File: src/organisms/dorel-section-content-block.html [hunk 2]
```diff
@@ -113,11 +121,29 @@ <h2 class="title">[[ blockData.title ]]</h2>
...
+       * checks the index of a component when used in a dom-repeat. Normal layout is
+       * image left text right
+       *
+       * @param {boolean} desktop breakpoint active or not
+       * @param {string} index of the repeated item
+       *
+       * @return {boolean}
+       */
       _reverseLayout(desktopBreakpoint, componentCount) {
+        console.log(desktopBreakpoint, componentCount);
```
Comment (aronmartin) [inline src/organisms/dorel-section-content-block.html]: Console?

File: src/organisms/dorel-section-content-block.html [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+       * @param {object} the complete blockdata object, containing the cta info
+       *
+       * @return {boolean} show the cta or not
+       */
+      _showCta(blockData) {
+        return (typeof blockData.cta_toggle !== 'undefined' && blockData.cta_toggle && blockData.cta_href && blockData.cta_text && blockData.cta_text !== '');
+      }
+    }
+
+    // Register the x-custom element with the browser
```
Comment (aronmartin) [inline src/organisms/dorel-section-content-block.html]: Maybe a bit to much documentation? the define customElements.define is quite self describing and part how you implement a polymer 2 component

Comment (mrpharderwijk) [reply src/organisms/dorel-section-content-block.html]: true its my own snippet ;)

File: src/organisms/dorel-section-content-block.html (hunk 1 above)
Comment (mrpharderwijk) [reply src/organisms/dorel-section-content-block.html]: and console i see ;)

**Pull request merged**